uikit/views/src/main/java/com/sygic/maps/uikit/views/navigation/lanes/data/SimpleLanesData.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ * This project is licensed under the MIT License.
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.uikit.views.navigation.lanes.data
+
+import com.sygic.maps.uikit.views.navigation.lanes.SimpleLanesView
+
+/**
+ * Data class holding lane data for views like [SimpleLanesView]
+ */
+data class SimpleLanesData(val directions: List<Int>, val highlighted: Boolean)
```

Comment:
Tu by možná bylo výstižnější highlight namísto highlighted.
```suggestion
data class SimpleLanesData(val directions: List<Int>, val highlight: Boolean)
```
potom by i lépe vypadal ten if:
```
if (lane.highlight) {
    set...
    set...
}
```

Reply:
co ja viem, mne to pride, ze to hovori o stave tych dat, ze su `higlighted`. Rovnako `LanesInfo` ma `isHighlighted` metodu

Reply:
mno oki